packages/rspack/src/util/bindingVersionCheck.ts

```diff
@@ -0,0 +1,18 @@
+import { execSync } from "child_process";
+
+function getVersion(packageName: string): string {
+	const command = `npm show ${packageName} version`;
```

Comment:
why not require version from package.json directly, npm show is very slow here

Reply:
> why not require version from package.json directly, npm show is very slow here

I'll change it